src/background.js

```diff
@@ -300,15 +325,44 @@ chrome.contextMenus.onClicked.addListener(async (info, tab) => {
 });
 
 // listen to keyboard shortcuts
-chrome.commands.onCommand.addListener(async (command) => {
+chrome.commands.onCommand.addListener(async (command, tab) => {
```

Comment:
`tab` argument is not availble on Firefox 🙄 🙄 🙄 

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/commands/onCommand

Reply:
Fixed

Reply:
`tab` is now available on Firefox 126

https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/126#changes_for_add-on_developers